tests/example-success/expected_results.json

```diff
@@ -1,4 +1 @@
-{
-  "version": 1,
-  "status": "pass"
-}
+{"version":2,"status":"pass","tests":[{"name":"leap.test.t1","test_code":"test.label \"leap should be false when year is not divisible by 4\" (test.expect ((leap 2015) === false))","status":"pass","message":"leap should be false when year is not divisible by 4 : Passed 1 tests."},{"name":"leap.test.t2","test_code":"test.label \"leap should be false when year is divisible by 2 not divisible by 4\" (test.expect ((leap 1970) === false))","status":"pass","message":"leap should be false when year is divisible by 2 not divisible by 4 : Passed 1 tests."},{"name":"leap.test.t3","test_code":"test.label \"leap should be true when year is divisible by 4\" (test.expect ((leap 1964) === true))","status":"pass","message":"leap should be true when year is divisible by 4 : Passed 1 tests."},{"name":"leap.test.t4","test_code":"test.label \"leap should be true when year divisible by 4, not divisible by 100\" (test.expect ((leap 1996)=== true))","status":"pass","message":"leap should be true when year divisible by 4, not divisible by 100 : Passed 1 tests."},{"name":"leap.test.t5","test_code":"test.label \"leap should be true when year divisible by 4 and 5\" (test.expect ((leap 1960)=== true))","status":"pass","message":"leap should be true when year divisible by 4 and 5 : Passed 1 tests."},{"name":"leap.test.t6","test_code":"test.label \"leap should be false when year divisible by 100, not divisible by 400\" (test.expect ((leap 2100)=== false))","status":"pass","message":"leap should be false when year divisible by 100, not divisible by 400 : Passed 1 tests."},{"name":"leap.test.t7","test_code":"test.label \"leap should be true when year divisible by 400 and 100\" (test.expect ((leap 2000) === true))","status":"pass","message":"leap should be true when year divisible by 400 and 100 : Passed 1 tests."},{"name":"leap.test.t8","test_code":"test.label \"leap should be false when year divisible by 4 and 100\" (test.expect ((leap 1900) === false))","status":"pass","message":"leap should be false when year divisible by 4 and 100 : Passed 1 tests."},{"name":"leap.test.t9","test_code":"test.label \"leap should be false when year divisible by 200, not divisble by 400\" (test.expect ((leap 1800) === false))","status":"pass","message":"leap should be false when year divisible by 200, not divisble by 400 : Passed 1 tests."}]}
```

Comment:
The `message` field for successful tests should be `null` or omitted. See https://exercism.org/docs/building/tooling/test-runners/interface